### PR TITLE
Rmk66:  2 little glitches, plus GITFNS prc dealing better with forking

### PR DESCRIPTION
**Pull request by rmkaplan**

### CODE REVIEW

**Review by nbriggs — COMMENTED** [review]
In the VT100SAVE record, the CHARSET field never seems to be referenced, so does it need to be there?  If it's referenced by some other code outside of VTCHAT then presumably those should be updated in this revision to use the new name.

**Comment by rmkaplan** [comment]
I did a scan and didn’t see it, but maybe there was a rationale.  So, convervatively, I at least left the cell.

File: library/VTCHAT [hunk 1]
```diff
@@ -151,7 +153,7 @@ Copyright (c) 1983-1988, 1990, 1993, 2022 by Venue & Xerox Corporation.
 
 (DECLARE%: EVAL@COMPILE
 
-(RECORD VT100SAVE (CURSORPOS CHARATTR CHARSET))
+(RECORD VT100SAVE (CURSORPOS CHARATTR VTSCHARSET))
```
Comment (masinter) [inline library/VTCHAT:157]: the CHARSET/VTSCHARSET field doesn't seem to be used. In ALLCONTENT.

**Review by masinter — APPROVED** [review]
After reallizing that I'd wandered into the second half of the PRC differences, LGTM

**Pull request merged**